TestLoopX/test/testloopx_outermost_interior.par

```diff
@@ -14,19 +14,39 @@ CarpetX::poison_undefined_values = "yes"
 
 CarpetX::xmin = -10
 CarpetX::ymin = -10
-CarpetX::zmin = -10
+CarpetX::zmin = 0
 CarpetX::xmax = +10
 CarpetX::ymax = +10
 CarpetX::zmax = +10
 
+CarpetX::blocking_factor_x = 2
```

Comment:
Here, I had set `blocking_factor` to 2 in order to make the number of cells divisible by the blocking factor.  However, I agree with you that the test parameter files should be kept simple.

So instead, I am proposing to change `ncells` to be divisible by the default blocking factor (8). I am also changing the grid extent so that the same spacing is kept the same as before (0.5), not that it matters for this test but purely for aesthetic reasons.